opencog/reasoning/pln/rules/inheritance-to-member.scm

```diff
@@ -0,0 +1,51 @@
+; =============================================================================
+; InheritanceToMemberRule
+;	InheritanceLink( B C )
+;			becomes
+;	MemberLink( B C )
+; -----------------------------------------------------------------------------
+
```

Comment:
I guess we want the rule with the { } in there for ExtensionalInheritance,
and probably the rule without the {} for IntensionalInheritance...

On Mon, May 11, 2015 at 4:04 PM, ngeiswei <notifications@github.com> wrote:

> In opencog/reasoning/pln/rules/inheritance-to-member.scm
> <https://github.com/opencog/opencog/pull/1593#discussion_r30019192>:
>
> > @@ -0,0 +1,51 @@
> > +; =============================================================================
> > +; InheritanceToMemberRule
> > +;	InheritanceLink( B C )
> > +;			becomes
> > +;	MemberLink( B C )
> > +; -----------------------------------------------------------------------------
> > +
>
> Is it
>
> InheritanceLink B C
> |-
> MemberLink B C
>
> or
>
> InheritanceLink {B} C
> |-
> MemberLink B C
>
> ?
>
> As the code shows below, the former is implemented. IMO the only way to
> justify this is via using the intensional side of the inheritance, but
> clearly given the formula implemented that is not what is meant.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencog/opencog/pull/1593/files#r30019192>.
>



-- 
Ben Goertzel, PhD
http://goertzel.org

"The reasonable man adapts himself to the world: the unreasonable one
persists in trying to adapt the world to himself. Therefore all progress
depends on the unreasonable man." -- George Bernard Shaw